lua/neoterm.lua

```diff
@@ -59,28 +103,62 @@ function neoterm.open(opts)
   if win_is_open() ~= true then
     local ui = vim.api.nvim_list_uis()[1]
 
-    local mode = opts.mode or config.mode
+    local position = opts.position or config.position
+    local width = opts.width or config.width
+    local height = opts.height or config.height
 
     local winopts = {
       relative = "editor",
-      width = math.floor(ui.width / 2),
-      height = ui.height - vim.o.cmdheight - 3,
-      row = 0,
-      col = ui.width,
       style = "minimal",
       border = "single",
     }
-    if mode == "horizontal" then
+
+    if type(position) == "string" then
+      position = ({
+        ["full screen"] = 0,
```

Comment:
I would prefer this stayed as `fullscreen`. Comments above and the error message would need to be updated as well

```suggestion
        ["fullscreen"] = 0,
```

Reply:
eb20ccc02c405458875c3f63a047b325476eac9e fixes the "position" type warning.

Also 59c299a3224886d141382de40791b87481494714 renames "full screen" to "fullscreen"